<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="invertMouseWheel">
           <property name="text">
            <string>Inverted mouse wheel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Snapping distance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="snappingDistanceLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sketch size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sketchSizeLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Point size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pointSizeLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Line shape size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineShapeSizeLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Angle units</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="angleUnitBox">
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>New file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="newFileEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Open file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="openFileEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Save file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="saveFileEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Save file as</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="saveAsFileEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Export file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="exportFileEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Print file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="printFileEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Settings </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="settingsEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="quitEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
